react/features/base/conference/actions.js

```diff
@@ -125,13 +125,14 @@ function _addConferenceListeners(conference, dispatch) {
     conference.on(
         JitsiConferenceEvents.DISPLAY_NAME_CHANGED,
         (id, displayName) => dispatch(participantUpdated({
+            conference,
```

Comment:
(I'm putting this comment here, but it applies to this entire commit).

Why do we need to pass the conference when updating a participant? Are we worried that the participant ID might collide? So, assuming that:

- participant IDs cannot collide
- a participant will always be added before it's updated

I don't see why we need the conference for PARTICIPANT_UPDATED. We can just get the participant by ID and the conference will be there. And it's not like that value can change.

Reply:
Participant IDs can collide across JitsiConference instances, participant IDs are "unique" only within a JitsiConference instance. That's why the identification of a remote participant needs to be the pair id-conference. Which is a lot bigger task than this PR, this PR reduces the number of places in which collisions can have lasting, visible effects on the app.

Reply:
👍